src/Classes/ServiceAPI/MyRadio_User.php

```diff
@@ -1594,6 +1693,28 @@ public function getEditForm()
                 )
             );
         }
+        $form->addField(
+            new MyRadioFormField(
+                'data_removal',
+                MyRadioFormField::TYPE_CHECK ,
+                [
+                'required' => false,
+                'label' => 'I Do not wish for my data to be deleted at any point. (this can be changed later)',
+                'options' => ['checked' => $this->hasOptedOutDeletion()],
+                ]
+            )
+        );
+        $form->addField(
+            new MyRadioFormField(
+                'hide',
+                MyRadioFormField::TYPE_CHECK ,
+                [
+                'required' => false,
+                'label' => 'Hide profile on public website (Strongly discouraged for current members)',
```

Comment:
is it even an option for current members? say, "hide public profile on website after x amount of time from last show?"

Reply:
I kind of want to leave it as an option for current members. If someone decides to leave the society half way through a year or has valid reasons for wanting a hidden profile I think it should be an option, however as said here a highly discouraged one.

Reply:
👍